CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing to fzf-eip 🔍 ♢
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## Workflow
+
+We use [Github Flow](https://guides.github.com/introduction/flow/index.html).
+
+All code changes happen through pull requests. They are the best way to propose changes to the codebase.
+
+We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+7. Review notes from team and update PR accordingly
+8. Once PR has been approved, the branch is deleted.
+
+## Contributions and License
+
+By contributing, you agree that your contributions will be licensed under its MIT License.
+
+Any contributions you make will be under the MIT Software License.
+
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Reporting Bugs
+
+Report bugs using Github [issues](https://github.com/cds-amal/fzf-eip/issues).
+
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/cds-amal/fzf-eip/issues). It's super easy!
+
+### Bug reports format
+
+Write bug reports with detail, background, and sample code.
+
+[This is an example](http://stackoverflow.com/q/12488905/180626) of a bug report written by someone else. We also think it's not a bad model. Here's [another example from Craig Hockenberry](http://www.openradar.me/11905408), another app developer.
+
```

Comment:
This is too complicated for this project

Reply:
Sweet, will remove.